package.json

```diff
@@ -7,7 +7,9 @@
     "develop": "strapi develop",
     "start": "strapi start",
     "build": "strapi build",
-    "strapi": "strapi"
+    "strapi": "strapi",
+    "test": "jest --runInBand",
```

Comment:
Can we mention how to run the test sin README.

Reply:
Added